src/igvfd/tests/data/inserts/differentiated_tissue.json

```diff
@@ -13,6 +13,7 @@
         "post_differentiation_time_units": "day",
         "treatments": [
             "4fbb0dc2-c72e-11ec-9d64-0242ac120002"
-        ]
+        ],
+        "differentiated_from": "/cell-lines/467c72a2-4f84-2c8f-96b0-ec8715e18185/"
```

Comment:
OK, this is a big question.  Thinking about the samples you were trying to post, do you think that they will want to register the specific H9 sample or do you think that this should point to an ontology term like H9.

Reply:
I think encode points to a specific H9 sample that all use the same donor. This cell line sample may have some info that can't be provided on the subsequent differentiated sample? Thinking about it I don't really see the point of this redundancy though, but someone with more experience on encode would probably provide more insight. I think it does make more sense to point to the ontology term for the derived cell line.

Reply:
Let's ask @jenjou and @gabdank I am thinking that we should consider differentiated from as an ontology term rather than a specific biosample, but I think those two would have more recent experience.  Also, I think if we add host in the future we should have that be an ontology term as well.

Reply:
Linking to a specific biosample could allow the labs to indicate additional metadata such as the starting quantity of cells, or provide a way to link a small group of biosamples originating from the same starting cells and differentiated into different cell fates. However, I'm not sure how often users would search for that type of info.

Reply:
i think we should link to biosample, rather than ontology. linking to ontology suffers from the downside cellosaurus suffers from - you are basically required to register as ontology term every genetically modified cell line, which I think we want to avoid (unlike cellosaurus).  i don't think differentiated_from is a required property and should be used to point to the sample it was differentiated from if such is available